packages/astro-utils/package.json

```diff
@@ -7,6 +7,9 @@
     ".": "./src/index.ts",
     "./sidebar": "./src/sidebar/index.ts",
     "./shiki": "./src/shiki/index.ts",
+    "./components/*": "./src/components/*",
+    "./utils/*": "./src/utils/*.ts",
+    "./css/*": "./src/css/*",
     "./expressive-code/*": "./dist/expressive-code/*.js"
   },
   "files": [
```

Comment:
Just noticed though this didn't seem to be a problem - but should this include the dist/expressive-code folder too?

Reply:
ah yeah id doesn't matter I guess as we don't publish it